### PR TITLE
Adds updates to week 1 readme schedule

### PR DESCRIPTION
**Pull request by Jen-Harris**
relates #656

Re-arranging order of readme schedule to be aligned with updated week plans.

### CODE REVIEW

File: coursebook/week-1/README.md [hunk 1]
```diff
@@ -6,22 +6,20 @@
...
 - 10:30 - 11:30 — Name game with as many members of Founders and Coders as possible!
 - 11:30 - 11:40 - Toilet/Coffee Break
-- 11:40 - 12:10 - Course overview
+- 11:40 - 13:00 - Course overview
     - [Code of Conduct](https://github.com/foundersandcoders/master-reference/blob/master/code_of_conduct.md)
     - [House rules](../general/house-rules.md)
     - Campus-specific info
     - Introduce gitter channels
-- 12:10 - 13.00 — Introduction to [Pair Programming](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/pair-programming.md)
+    - Go through 16 week [schedule](https://github.com/foundersandcoders/master-reference/tree/master/coursebook)
```
Comment (jsms90) [inline coursebook/week-1/README.md]: I think this should probably be the first thing that is introduced, before jumping into the rest? i.e. where it says "course overview" on line 13

I would think of the schedule as the "What" of the programme & the Code of Conduct and House Rules as the "How".

Then the campus specific stuff after that?

What do you think @Jen-Harris ?

File: coursebook/week-1/README.md [hunk 2]
```diff
@@ -32,8 +30,8 @@
 - 12:45 - 13:00 — Introduce [Research topics](./research-afternoon.md)
 
 — LUNCH —
-- 14.00 - 16:30 — Research
-- 16:30 - 17:30 — Review and Present research topics
+- 14.00 - 16:45 — Research / TBC
```
Comment (jsms90) [inline coursebook/week-1/README.md]: Probably not worth changing in the schedule until we have something specific to change it to?

File: coursebook/week-1/README.md [hunk 3]
```diff
@@ -32,8 +30,8 @@
...
+- 16:45 - 17:30 — Talk on consensus based decision-making
```
Comment (jsms90) [inline coursebook/week-1/README.md]: My understanding was that we were changing this from a talk to a more interactive demonstration of our process?

File: coursebook/week-1/README.md [hunk 4]
```diff
@@ -51,13 +49,15 @@
...
 
-- 14:00-15:00 -[Cohort Stop Go Continue (retrospective)](./retrospectives.md#cohort-retrospective)
-- 15:00-15:40 - [Team retrospectives](./retrospectives.md#team-retrospective)
-- 15:40-17:00 - Presentations (20 mins per group)
-- 17:00-18:00 - External Speaker or Alumni Project Presentation
+- 14:00 - 14.15 — [Plan presentations](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/weekly-projects.md#project-presentation)
+- 14:15 - 15:35 - Presentations (20 mins per group)
+- 15:35 - 15:45 - Toilet/Coffee Break
+- 15:45 - 16:45 -[Cohort Stop Go Continue (retrospective)](./retrospectives.md#cohort-retrospective)
+- 16:45 - 17:00 - [Team retrospectives](./retrospectives.md#team-retrospective)
```
Comment (jsms90) [inline coursebook/week-1/README.md]: I'm not sure that this is sufficient time for a retrospective. @eliascodes @des-des @mantagen what do you think?

Could "plan presentations" stay before lunch, if we do presentations directly after lunch?

File: coursebook/week-1/README.md [hunk 5]
```diff
@@ -51,13 +49,15 @@
 - 14.00 - 18.00 — Projects
 
 ## Day 5
-- 10.00 - 11.00 — [Code review](../general/code-reviews.md) (1 alumnus per project)
-- 11.00 - 12.45 — Respond to issues
-- 12.45 - 13.00 — [Plan presentations](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/weekly-projects.md#project-presentation)
+- 10:00 - 10:15 - How to Code review
```
Comment (jsms90) [inline coursebook/week-1/README.md]: Link to `../general/code-reviews.md` here, rather than the slot below?

**Comment by Jen-Harris** [comment]
@jsms90 I've made the updates. Please can you review again? 👍

**Comment by jsms90** [comment]
@Jen-Harris the "how to code review" link can prob be changed in PR #662 

I don't know what the other Course Facilitators think, but Team Retrospectives seem to be the least well thought out part of day 5, so maybe limiting it to 15 minutes isn't the end of the world anyway. Happy to merge :+1:

File: coursebook/week-1/README.md (hunk 2 above)
Comment (veredrec) [reply coursebook/week-1/README.md]: @jsms90 I agree!

**Pull request merged**